cmd/node/flags.go

```diff
@@ -523,6 +523,19 @@ func getWorkingDir(ctx *cli.Context, cliFlag cli.StringFlag, log logger.Logger)
 	return workingDir
 }
 
+func getCustomDirIfSet(ctx *cli.Context, cliFlag cli.StringFlag, log logger.Logger) string {
+	dbDir := ctx.GlobalString(cliFlag.Name)
+
+	var dirStr string
+	if len(dbDir) == 0 {
```

Comment:
[optional]
could have been simplified to:
```
if len(dbDir) == 0 {
   return getWorkingDir(ctxm workingDirectory, log)
}

return getWorkingDir(ctx, cliFlag, log)
```

Reply:
done